code/game/gamemodes/wizard/wizard.dm

```diff
@@ -9,162 +9,33 @@
 	required_players = 20
 	required_enemies = 1
 	recommended_enemies = 1
-	enemy_minimum_age = 14
+	enemy_minimum_age = 0

```

Comment:
I'd also suggest a minimum age here.  Wizard has one main antag that if dead, ends the round.  Because of this, we'll want someone who at least knows the basics of SS13.

Now, as far as what the minimum age should be, I am okay with 3 days.